pyccel/ast/datatypes.py

```diff
@@ -100,6 +100,7 @@
                                  'complex':('complex',default_precision['complex']),  # to create numpy array with dtype='complex'
                                  'complex64':('complex',4),
                                  'complex128':('complex',8),
+                                 'bool' :('bool',1),
```

Comment:
we are already setting `bool` precision [here](https://github.com/pyccel/pyccel/pull/776/files#diff-62efbcaf36568e28d52d7aa8d3d25d515157238dd4bff069483b0e8778cf9a9eR111)